system/lib/pthread/library_pthread.c

```diff
@@ -247,11 +247,11 @@ int pthread_getattr_np(pthread_t t, pthread_attr_t *a)
 
 static uint32_t dummyZeroAddress = 0;
 
-int usleep(unsigned usec)
+static void do_sleep(double msecs)
 {
 	int is_main_thread = emscripten_is_main_runtime_thread();
 	double now = emscripten_get_now();
-	double target = now + usec * 1e-3;
```

Comment:
Btw, the old code here is horribly wrong. The multiplication wanted to convert microseconds to milliseconds (1000 microseconds = 1 millisecond so divide by 1000), but instead converted microseconds to nanoseconds by multiplying by 1000, so looks like sleeping was broken in pthreads builds as well(!)